tests/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Commands/DeleteAllCountryLocalNamesForCountryCommand.g.cs

```diff
@@ -0,0 +1,70 @@
+﻿﻿﻿// Generated
+
+#nullable enable
+
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Nox.Application.Commands;
+using Nox.Solution;
+using Nox.Domain;
+using Nox.Types;
+using Nox.Application.Factories;
+using Nox.Exceptions;
+using ClientApi.Domain;
+using ClientApi.Application.Dto;
+using Dto = ClientApi.Application.Dto;
+using CountryEntity = ClientApi.Domain.Country;
+
+namespace ClientApi.Application.Commands;
+
+public partial record DeleteAllCountryLocalNamesForCountryCommand(CountryKeyDto ParentKeyDto, System.Guid? Etag) : IRequest <bool>;
+
+
+internal partial class DeleteAllCountryLocalNamesForCountryCommandHandler : DeleteAllCountryLocalNamesForCountryCommandHandlerBase
+{
+	public DeleteAllCountryLocalNamesForCountryCommandHandler(
+        IRepository repository,
+		NoxSolution noxSolution)
+		: base(repository, noxSolution)
+	{
+	}
+}
+
+internal partial class DeleteAllCountryLocalNamesForCountryCommandHandlerBase : CommandBase<DeleteAllCountryLocalNamesForCountryCommand, CountryEntity>, IRequestHandler <DeleteAllCountryLocalNamesForCountryCommand, bool>
+{
+	public IRepository Repository { get; }
+
+	public DeleteAllCountryLocalNamesForCountryCommandHandlerBase(
+        IRepository repository,
+		NoxSolution noxSolution) : base(noxSolution)
+	{
+		Repository = repository;
+	}
+
+	public virtual async Task<bool> Handle(DeleteAllCountryLocalNamesForCountryCommand request, CancellationToken cancellationToken)
+	{
+		cancellationToken.ThrowIfCancellationRequested();
+		await OnExecutingAsync(request);
+		
+		var keys = new List<object?>(1);
+		keys.Add(Dto.CountryMetadata.CreateId(request.ParentKeyDto.keyId));
```

Comment:
including localized should be unnecessary. This is being setup to cascade delete
we want to "hide" the localized concept from our domain.

For example when deleting a single owned entity we are not doing it, so the related localized is being deleted

see NoxDatabaseConfigurator for Localized Entity

Reply:
Desired changes have been applied